libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -80,7 +80,7 @@ const static uint16_t   default_max_inline_action_depth              = 4;
 const static uint16_t   default_max_auth_depth                       = 6;
 const static uint32_t   default_sig_cpu_bill_pct                     = 50 * percent_1; // billable percentage of signature recovery
 const static uint32_t   default_produce_block_offset_ms              = 450;
-const static uint32_t   default_production_pause_vote_timeout_ms     = 6u*1000u; // 6 seconds
+const static uint32_t   default_production_pause_vote_timeout_ms     = 13u*1000u; // 13 seconds
```

Comment:
Maybe add a comment here why 13 is chosen

Reply:
Updated to new algorithm. Reverted back to 6 seconds.